modules/mobile/app/controllers/mobile/v0/awards_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'bgs/awards_service'
+require 'bid/awards/service'
+
+module Mobile
+  module V0
+    class AwardsController < ApplicationController
+      def index
+        award_data = regular_award_service.get_awards
+        award_data.merge!(pension_award_service.get_awards_pension.body['awards_pension']&.transform_keys(&:to_sym))
+        award_data[:id] = current_user.uuid
+        awards = Mobile::V0::Award.new(award_data)
```

Comment:
So does this only create a single record? The specs and docs seem to imply that this only returns a single record, which is consistent with what you're doing here. But this is an index method that calls `get_awards` and the object here is called `awards`, which I guess is kinda reasonable since it's two awards smashed into one record. It's a little surprising that `get_awards` only returns one record and you don't have to do `award_data.first.merge`.

Reply:
I was really going back and forth on this. I agree with you, index implies a list, but this does not return a list. However, I am merging two awards into a single object. It's unfortunate the service method is called `get_awards` and not `get_award_overview` or something like that because it just returns a single object with a bunch of attributes.

If you'd like me to move it to `#show` I could easily see that argument. I left it in index because it was merging two responses and the single object encompasses multiple awards, it's just combined for some unknown reason

Reply:
No need to change it. It's a bit of a gray area and really has no impact that I can think of. I think one could argue that a show should have an id in the route, which this does not, so 🤷.